content/integrations/activemq.md

```diff
@@ -105,7 +105,7 @@ else echo -e "\e[031mAgent is not running\e[0m"; fi
 ## Data Collected
 ### Metrics
 
-{{< get-metrics-from-git "activemq_xml" >}}
+{{< get-metrics-from-git >}}
```

Comment:
activemq does not have metrics here: https://github.com/DataDog/integrations-core/blob/master/activemq/metadata.csv

this may cause the build to abort

Reply:
👍  good catch

Reply:
The git integration title param is activemq_xml, so  it's pulling the right metadata.csv :)